api/reference/1.0/secrets/create.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Create
+---
+
+# Create a Secret
+
+Add a Secret to a Property. 
+
+When you create a new secret, you will get an immediate response that contains information for that resource.  
+
+At the same time, a secret exchange task is triggered to test that the credential exchange is functional.  This task is processed asynchronously and updates the secret's `status` attribute with one of two possible values:
+
+#### `succeeded`
+
+This is the status when the exchange request for the secret completes successfully.
+
+#### `failed`
+
+This is the status when the exchange request for the secret is not successful.
+
```

Comment:
I think we should provide information to the user for failed secrets.  Where will they get information on what went wrong?

Reply:
I added the `api/guides/secrets.md`. Can you check it out?